### PR TITLE
 [Remove Spring extender usages] LPS-159403 

### PR DESCRIPTION
**Pull request by JanisZhang**
Hi @dantewang,

This PR is to remove spring extender usage in commerce-wish-list-service, thanks!
https://issues.liferay.com/browse/LPS-159403

### CODE REVIEW

**Comment by JanisZhang** [comment]
Hi @catarineaguiar , Could you review the test results when they finish? Thank you!

**Comment by Kvale1733** [comment]
@JanisZhang No regressions found.

File: modules/apps/commerce/commerce-wish-list-service/src/main/java/com/liferay/commerce/wish/list/service/impl/CommerceWishListLocalServiceImpl.java [hunk 1]
```diff
@@ -348,7 +347,4 @@ protected void validateGuestWishLists() throws PortalException {
 	@Reference
 	private CommerceWishListItemLocalService _commerceWishListItemLocalService;
 
-	@Reference
-	private DDMFormValuesHelper _ddmFormValuesHelper;
```
Comment (dantewang) [inline modules/apps/commerce/commerce-wish-list-service/src/main/java/com/liferay/commerce/wish/list/service/impl/CommerceWishListLocalServiceImpl.java:340]: @JanisZhang Could you look into the git history to see why this was added? If it was added with usages, but then was forgotten when the usages were removed, then it's safe to remove this.

**Review by dantewang — COMMENTED** [review]
@JanisZhang Code looks good to me, but please double check as I commented. Thanks!

File: modules/apps/commerce/commerce-wish-list-service/src/main/java/com/liferay/commerce/wish/list/service/impl/CommerceWishListLocalServiceImpl.java (hunk 1 above)
Comment (JanisZhang) [reply modules/apps/commerce/commerce-wish-list-service/src/main/java/com/liferay/commerce/wish/list/service/impl/CommerceWishListLocalServiceImpl.java:340]: Hi @dantewang , `_ddmFormValuesHelper` was replaced by `Objects.equals  ` in 23ec8cfe9df7ea0c8e727c277264dcb686d54b28, it's safe to delete. Thanks!

**Review by dantewang — APPROVED** [review]

**Comment by tinatian** [comment]
@JanisZhang Please see https://github.com/liferay-core-infra/liferay-portal/pull/1292 and resend.

Tina

**Pull request closed without merging**